tests/clickhouse-test

```diff
@@ -585,6 +585,10 @@ class TestCase:
         self.stderr_file = (
             os.path.join(suite.suite_tmp_path, self.name) + file_suffix + ".stderr"
         )
+        # Printed only in case of failures
+        self.debug_log_file = (
+            os.path.join(suite.suite_tmp_path, self.case_file) + file_suffix + ".log"
```

Comment:
Great, CI founds that `file_suffix` was not used in `*.expect` - https://s3.amazonaws.com/clickhouse-test-reports/38728/fdebae1f72849f8896b11a35f68953f48abc3e00/stateless_tests_flaky_check__address__actions_.html

Rebased and fixed.